commit 07cf8aa922db7747cd6e100d2e3f7ca839c7a419
Author: Will Deacon <will@kernel.org>
Date:   Fri Nov 13 11:38:41 2020 +0000

    KVM: arm64: Make BP hardening globals static instead
    
    Branch predictor hardening of the hyp vectors is partially driven by a
    couple of global variables ('__kvm_bp_vect_base' and
    '__kvm_harden_el2_vector_slot'). However, these are only used within a
    single compilation unit, so internalise them there instead.
    
    Signed-off-by: Will Deacon <will@kernel.org>
    Signed-off-by: Marc Zyngier <maz@kernel.org>
    Cc: Marc Zyngier <maz@kernel.org>
    Cc: Quentin Perret <qperret@google.com>
    Link: https://lore.kernel.org/r/20201113113847.21619-5-will@kernel.org

diff --git a/arch/arm64/kvm/va_layout.c b/arch/arm64/kvm/va_layout.c
index e0404bcab019..d1195c288c9f 100644
--- a/arch/arm64/kvm/va_layout.c
+++ b/arch/arm64/kvm/va_layout.c
@@ -131,9 +131,6 @@ void __init kvm_update_va_mask(struct alt_instr *alt,
 	}
 }
 
-void *__kvm_bp_vect_base;
-int __kvm_harden_el2_vector_slot;
-
 void kvm_patch_vector_branch(struct alt_instr *alt,
 			     __le32 *origptr, __le32 *updptr, int nr_inst)
 {